### PR TITLE
Adds location for TSconfig

### PR DESCRIPTION
**Pull request by funkvn**
As a result of folder name/file extension desicion:
https://decisions.typo3.org/t/file-endings-for-typoscript-and-tsconfig-files-results/71

Relates: https://forge.typo3.org/issues/81686

### CODE REVIEW

File: Documentation/ExtensionArchitecture/FilesAndLocations/Index.rst [hunk 1]
```diff
@@ -274,6 +274,11 @@ Classes/Domain/Repository
 Classes/ViewHelpers
   Helper classes used in the views.
 
+Configuration/TsConfig/Page
+Configuration/TsConfig/User
+  Page and User TSconfig, see `TSconfig Reference <https://docs.typo3.org/typo3cms/TSconfigReference/>`_.
+  Files must have the file extension :file:`.tsconfig`.
```
Comment (liayn) [inline Documentation/ExtensionArchitecture/FilesAndLocations/Index.rst]: should have

**Comment by funkvn** [comment]
Hi at all. I was not sure how to handle such "defaults". I'm prefer the more strict variants. :-)

**Comment by marble** [comment]
@funkvn I'm not sure what you mean. Personally I've started using '.ts.txt'

**Comment by funkvn** [comment]
@marble I do not understand why TYPO3 should handle more than one file extension for the same content. Therefore I understood the "default" file extension a the one and only.

I know this will be breaking, but why not cleaning up this for v9?

**Pull request merged**